phoenicis-javafx/src/main/java/com/playonlinux/javafx/views/mainwindow/settings/ViewSettings.java

```diff
@@ -125,15 +125,9 @@ public String fromString(String string) {
     }
 
     public void setSettings(Settings settings) {
-        if (settings.get(Setting.THEME).equals("dark")) {
-            themes.setValue(Themes.DARK);
-        } else if (settings.get(Setting.THEME).equals("breeze")) {
-            themes.setValue(Themes.BREEZE_DARK);
-        } else if (settings.get(Setting.THEME).equals("hidpi")) {
-            themes.setValue(Themes.HIDPI);
-        } else {
-            themes.setValue(Themes.DEFAULT);
-        }
+        final Themes setTheme = Themes.fromShortName(settings.get(Setting.THEME));
```

Comment:
Could we use enum values instead of Strings? e.g.
```java
settings.getTheme(Setting.THEME)
```
which would return an enum.

Could also be useful for other data types (e.g. `settings.getDouble()`).